src/main/routes/course.js

```diff
@@ -107,9 +110,22 @@ const course_new_page = async (res, department = false) => {
 /* GET course home page */
 router.route('/')
 	.get(html.auth_wrapper(async (req, res, next) => {
+		const artifacts_active 			=  
+			{ arti : await Artifact.query().eager('[evaluations, owner.owner.[semester,owner.owner]]'),
```

Comment:
No space after `arti` to be consistent with the rest of the codebase, and name should maybe be complete (`artifact`).